source/qml_v2/AdaptiveCardQmlEngine/qml/ComboboxRender.qml

```diff
@@ -0,0 +1,174 @@
+        
+        width: comboBoxConstants.indicatorWidth
+        horizontalPadding: comboBoxConstants.arrowIconHorizontalPadding
+        verticalPadding: comboBoxConstants.arrowIconVerticalPadding
+        icon.width: comboBoxConstants.arrowIconWidth
+        icon.height: comboBoxConstants.arrowIconHeight
+        focusPolicy: Qt.NoFocus
+        icon.color: comboBoxConstants.arrowIconColor
+        icon.source: comboBox.popup.visible ? "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iOCIgdmlld0JveD0iMCAwIDE0IDgiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxwYXRoIGQ9Ik0xMy4zNTM2IDYuNjQ2NTRMNy4zNTM2MSAwLjY0NjU0QzcuMjU5ODUgMC41NTI3ODkgNy4xMzI2OCAwLjUwMDEyMiA3LjAwMDA5IDAuNTAwMTIyQzYuODY3NDkgMC41MDAxMjIgNi43NDAzMyAwLjU1Mjc4OSA2LjY0NjU2IDAuNjQ2NTRMMC42NDY1NjEgNi42NDY1NEMwLjU5OTE2NSA2LjY5Mjc1IDAuNTYxNDE3IDYuNzQ3OTEgMC41MzU1MDUgNi44MDg4MkMwLjUwOTU5NCA2Ljg2OTczIDAuNDk2MDM1IDYuOTM1MTggMC40OTU2MTYgNy4wMDEzOEMwLjQ5NTE5NiA3LjA2NzU3IDAuNTA3OTI0IDcuMTMzMTkgMC41MzMwNjIgNy4xOTQ0MkMwLjU1ODE5OSA3LjI1NTY2IDAuNTk1MjQ2IDcuMzExMjkgMC42NDIwNTIgNy4zNTgxQzAuNjg4ODU4IDcuNDA0OTEgMC43NDQ0OTMgNy40NDE5NSAwLjgwNTcyOCA3LjQ2NzA5QzAuODY2OTY0IDcuNDkyMjMgMC45MzI1ODEgNy41MDQ5NiAwLjk5ODc3NCA3LjUwNDU0QzEuMDY0OTcgNy41MDQxMiAxLjEzMDQyIDcuNDkwNTYgMS4xOTEzMyA3LjQ2NDY1QzEuMjUyMjQgNy40Mzg3MyAxLjMwNzQgNy40MDA5OSAxLjM1MzYxIDcuMzUzNTlMNy4wMDAwNiAxLjcwNzA5TDEyLjY0NjYgNy4zNTM1OUMxMi42OTI4IDcuNDAwOTkgMTIuNzQ3OSA3LjQzODczIDEyLjgwODggNy40NjQ2NUMxMi44Njk4IDcuNDkwNTYgMTIuOTM1MiA3LjUwNDEyIDEzLjAwMTQgNy41MDQ1NEMxMy4wNjc2IDcuNTA0OTYgMTMuMTMzMiA3LjQ5MjIzIDEzLjE5NDQgNy40NjcwOUMxMy4yNTU3IDcuNDQxOTUgMTMuMzExMyA3LjQwNDkxIDEzLjM1ODEgNy4zNTgxQzEzLjQwNDkgNy4zMTEyOSAxMy40NDIgNy4yNTU2NiAxMy40NjcxIDcuMTk0NDJDMTMuNDkyMiA3LjEzMzE5IDEzLjUwNSA3LjA2NzU3IDEzLjUwNDYgNy4wMDEzOEMxMy41MDQxIDYuOTM1MTggMTMuNDkwNiA2Ljg2OTczIDEzLjQ2NDcgNi44MDg4MkMxMy40Mzg4IDYuNzQ3OTEgMTMuNDAxIDYuNjkyNzUgMTMuMzUzNiA2LjY0NjU0WiIgZmlsbD0iYmxhY2siIGZpbGwtb3BhY2l0eT0iMC45NSIvPgo8L3N2Zz4K" : "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iOCIgdmlld0JveD0iMCAwIDE0IDgiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxwYXRoIGQ9Ik0xMy4zNTM3IDAuNjQ2NTRDMTMuMjYgMC41NTI3ODkgMTMuMTMyOCAwLjUwMDEyMiAxMy4wMDAyIDAuNTAwMTIyQzEyLjg2NzYgMC41MDAxMjIgMTIuNzQwNSAwLjU1Mjc4OSAxMi42NDY3IDAuNjQ2NTRMNi45OTk5OSA2LjI5MzA0TDEuMzUzNDkgMC42NDY1NEMxLjI1OTI1IDAuNTU1MTI5IDEuMTMyODQgMC41MDQ0NTQgMS4wMDE1NSAwLjUwNTQ1N0MwLjg3MDI2NCAwLjUwNjQ2IDAuNzQ0NjM5IDAuNTU5MDYxIDAuNjUxODA1IDAuNjUxOTAxQzAuNTU4OTcyIDAuNzQ0NzQxIDAuNTA2Mzc5IDAuODcwMzcgMC41MDUzODUgMS4wMDE2NkMwLjUwNDM5MiAxLjEzMjk0IDAuNTU1MDc2IDEuMjU5MzYgMC42NDY0OTQgMS4zNTM1OUw2LjY0NjQ5IDcuMzUzNTlDNi43NDAyNiA3LjQ0NzMzIDYuODY3NDEgNy40OTk5OCA2Ljk5OTk5IDcuNDk5OThDNy4xMzI1OCA3LjQ5OTk4IDcuMjU5NzMgNy40NDczMyA3LjM1MzUgNy4zNTM1OUwxMy4zNTM1IDEuMzUzNTlDMTMuNDQ3MyAxLjI1OTg2IDEzLjUgMS4xMzI3MSAxMy41IDEuMDAwMTJDMTMuNTAwMSAwLjg2NzUyMyAxMy40NDc1IDAuNzQwMzM5IDEzLjM1MzcgMC42NDY1NFoiIGZpbGw9ImJsYWNrIiBmaWxsLW9wYWNpdHk9IjAuOTUiLz4KPC9zdmc+Cg=="
+        enabled: false
```

Comment:
What is this ? does it work correctly ?

Reply:
This is a comboboxArrowIcon and it is working